pkg/report/linux.go

```diff
@@ -147,6 +147,7 @@ func (ctx *linux) Parse(output []byte) (desc string, text []byte, start int, end
 		pos = next + 1
 	}
 	if oops == nil {
+		corrupted = isCorrupted("", string(text))
```

Comment:
I don't understand this part. If there is no oops/desc, we did not detect a crash. Why do we need corrupted flag in this case? And what's corrupted if there is no bug?

Reply:
We don't. You already removed this as far as I see.